CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+## CHANGELOG
+
+### 1.2.0 (2020-07-21)
+**SDK**
+- Upgraded build setup and some internal dependencies
+- Added support for customization of user agent information
+- Added feature to support backward & forward navigation support in a mini app
+- Added support for javascript of type module
```

Comment:
This isn't very clear what was fixed. Maybe add an example: `<script src="some_module.js" type="module"></script>`.

Reply:
Okay, we can give an example too.